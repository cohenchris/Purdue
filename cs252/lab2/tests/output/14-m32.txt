0x56684290
0x566840c8
[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0200
]

[
	addr: 0200
	size: 400
	left_size: 40
	allocated: false
	prev: 0656
	next: 0104
]

[
	addr: 0104
	size: 40
	left_size: 40
	allocated: false
	prev: 0200
	next: 0008
]

[
	addr: 0008
	size: 40
	left_size: 0
	allocated: false
	prev: 0104
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK
0x56684290
0x56684068
[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0104
]

[
	addr: 0104
	size: 40
	left_size: 40
	allocated: false
	prev: 0656
	next: 0008
]

[
	addr: 0008
	size: 40
	left_size: 0
	allocated: false
	prev: 0104
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK
0x56684290
0x56684008
[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0008
]

[
	addr: 0008
	size: 136
	left_size: 0
	allocated: false
	prev: 0656
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK
0x56684038
0x56684290
[
	addr: 0056
	size: 88
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0656
]

[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: 0056
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0656
	next: NULLPTR
]

BREAK
