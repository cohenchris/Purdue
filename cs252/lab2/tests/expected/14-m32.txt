

[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0200
]

[
	addr: 0200
	size: 400
	left_size: 40
	allocated: false
	prev: 0656
	next: 0104
]

[
	addr: 0104
	size: 40
	left_size: 40
	allocated: false
	prev: 0200
	next: 0008
]

[
	addr: 0008
	size: 40
	left_size: 0
	allocated: false
	prev: 0104
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK


[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0104
]

[
	addr: 0104
	size: 40
	left_size: 40
	allocated: false
	prev: 0656
	next: 0008
]

[
	addr: 0008
	size: 40
	left_size: 0
	allocated: false
	prev: 0104
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK


[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0008
]

[
	addr: 0008
	size: 136
	left_size: 0
	allocated: false
	prev: 0656
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0008
	next: NULLPTR
]

BREAK


[
	addr: 0056
	size: 88
	left_size: 40
	allocated: false
	prev: NULLPTR
	next: 0656
]

[
	addr: 0656
	size: 40
	left_size: 40
	allocated: false
	prev: 0056
	next: 4024
]

[
	addr: 4024
	size: 56
	left_size: 3312
	allocated: false
	prev: 0656
	next: NULLPTR
]

BREAK
